packages/serverless-contracts-plugin/src/plugin/utils/generateOpenApiDocumentation.ts

```diff
@@ -1,19 +1,21 @@
 import { OpenAPIV3 } from 'openapi-types';
 import Serverless from 'serverless';
 
-import { isInstanceOfDocumentedContract } from '@swarmion/serverless-contracts';
+import { getOpenApiDocumentation } from '@swarmion/serverless-contracts';
 
 import { ServerlessContracts } from 'types/serviceOptions';
 
+import { isDefined } from './isDefined';
```

Comment:
We can use lodash for this

Reply:
It doesn't seem to work for now, so I think we can keep it that way